veggieseasons/lib/data/model.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:scoped_model/scoped_model.dart';
+import 'package:veggieseasons/data/veggie.dart';
+import 'package:veggieseasons/data/local_veggie_provider.dart';
+
+class AppState extends Model {
+  List<Veggie> veggies;
+
+  AppState() : veggies = LocalVeggieProvider.veggies;
+
+  List<Veggie> get allVeggies => veggies;
```

Comment:
[nit] This seems like a remnant of something from the past. Why not just use `veggies` since it's public.

Reply:
Fixed by taking `veggies` private as `_veggies`.